docs/troubleshooting/common-problems.md

```diff
@@ -271,3 +271,30 @@ It should be empty, but if you have the bug, you got `check_update`.
 Remove `/var/lib/xcp-ng-xapi-plugins/updater.py.lock` and that should fix it.
 
 ---
+
+## Unable to live migrate VDI between SRs
+
+### Cause
```

Comment:
Let's add a `### Symptoms` section first. 

Update: so, that's something we should have applied to all sections in this file. Your addition is consistent, but our structure doesn't really make sense. We should always describe what is visibly wrong before getting into the details of the causes :D

So, forget about my comment for now, or add the description of the problem to all of them. Your choice.